dev-docs/source/RunningTheUnitTests.rst

```diff
@@ -103,31 +103,33 @@ Starting in your build folder (e.g. Mantid/Code/debug):
 
    -  Not possible with ctest.
 
-Visual Studio/ XCode note
-#########################
+Running Unit Tests from Within Visual Studio
+############################################
 
-In Visual Studio the user can alter the properties of the subset of
-tests (inside the unitTest directory (e.g. AlgorithmTest). In the
-properties box it is possible to specify a specific test to run by
-typing its name in the TargetName box. Then to execute the test, right
-click the subset of tests and select debug and then start new instance.
+Open the Mantid solution in Visual Studio 2015. To run a subset of tests (for example ``UnitTests/AlgorithmsTest``);
```

Comment:
Can we just say "Visual Studio" here and not be explicit about the version. When we move forward in the future it'd be easier no to have to track down more replaced explicitly referencing 2015.